ibm-mq-mixin/README.md

```diff
@@ -0,0 +1,126 @@
+# IBM MQ Mixin
+
+IBM MQ mixin is a set of configurable Grafana dashboards and alerts.
+
+The IBM MQ mixin contains the following dashboards:
+
+- IBM MQ cluster overview
+- IBM MQ queue overview
+- IBM MQ queue manager overview
+- IBM MQ topics overview
+
+and the following alerts:
+
+- IBMMQStaleMessages
+- IBMMQStaleMessages
+- IBMMQLowDiskSpace
+- IBMMQHighQueueManagerCpuUsage
+
+## IBM MQ cluster overview 
+
+The IBM MQ cluster overview dashboard provides details on the number and status of clusters, queue managers, queues, and topics within and IBM MQ cluster. This also includes information about the amount of time it takes for a messsage to get through a transmission queue.
+
+# screenshots
+![]()
```

Comment:
Speaking of screenshots, I think the ones in the PR need more load in order to come across better. I.e. more operations and an actual `mq_cluster` and not the `<mq_cluster>`

Reply:
I can for sure do more operations, but I will say in that screenshot, we are using an MQ cluster named `INVENTORY` . Given the screenshot is a little rough. We were aiming to retake some shots once we got some more eyes on the PR and if there was any more feedback.

Reply:
Would definitely appreciate a bit more load, shows up much nicer in our docs that way too :)